spec/metrics/3scale_backend_calls_spec.lua

```diff
@@ -0,0 +1,73 @@
+describe('backend calls metrics', function()
+  describe('report', function()
+    local threescale_backend_calls_metrics
+    local test_counter = { inc = function() end }
+
+    before_each(function()
+      stub(test_counter, 'inc')
+      local Prometheus = require('apicast.prometheus')
+      getmetatable(Prometheus).__call = function(_, type)
+        if type == 'counter' then
+          return test_counter
+        end
+      end
+
+      package.loaded['apicast.metrics.3scale_backend_calls'] = nil
```

Comment:
Similar comment as above. If we have to do this, looks like something is broken in our interface.